test/e2e/clone_test.go

```diff
@@ -61,15 +61,14 @@ func testPVCClone() {
 		}).Should(Succeed())
 
 		By("writing a file on mountpath")
-		writePath := "/test1/bootstrap.log"
 		Eventually(func() error {
-			_, err := kubectl("exec", "-n", nsCloneTest, "thinpod", "--", "cp", "/var/log/bootstrap.log", writePath)
+			_, err := kubectl("exec", "-n", nsCloneTest, "thinpod", "--", "cp", "/var/log/bootstrap.log", "/test1/bootstrap.log")
```

Comment:
I think this change is actually bad, and if a linter flags this, we should not adopt it like this.